data/constants/keycodes/keycodes_0.0.7.hjson

```diff
@@ -0,0 +1,7 @@
+{
+    "ranges": {
+        "0x77C0/0x003F": {
+            "define": "QK_COMMUNITY_MODULE"
```

Comment:
I see range `QK_COMMUNITY_MODULE` to `QK_COMMUNITY_MODULE_MAX` is reserved for modules, which is helpful. If multiple modules are used at once, and each needs a few keycodes, is there a way that each module could get part of this range without overlapping with other modules? It would be great if modules could each allocate keycodes in a standardized way, essentially, an automatic way to enumerate the range like:

```c
enum module_keycodes {
  // Keycodes for module Foo.
  FOO_KC1 = QK_COMMUNITY_MODULE,
  FOO_KC2,
  FOO_KC3,
  FOO_KC4,
  // Keycodes for module Bar.
  BAR_KC1,
  BAR_KC2,
  // Keycodes for more modules...
};
```

Reply:
All module keycodes are assigned automatically, with keyboard module keycodes preceding keymap module keycodes, in the module order specified in the keyboard or keymap json.